app/qml/pages/DeviceListPage.qml

```diff
@@ -224,7 +224,7 @@ Page {
             ColumnView {
                 id: trustedDevices
                 width: page.width
-                itemHeight: Theme.itemSizeMedium
+                itemHeight: Theme.itemSizeMedium+(3*Theme.itemSizeSmall/trustedDevices.count)
```

Comment:
Please insert spaces before and after each operator, as in the existing of the code.

Reply:
Done